src/snowflake/cli/_plugins/nativeapp/entities/application_package.py

```diff
@@ -915,6 +916,7 @@ def validate_setup_script(
             if validation_result["status"] == "FAIL":
                 raise SetupScriptFailedValidation()
 
+    @start_cli_metrics_span("get_validation_result")
```

Comment:
ditto, let's not name things according to the implementation. That can change over time.